app/lib/simulator/Simulator.coffee

```diff
@@ -95,7 +95,8 @@ module.exports = class Simulator extends CocoClass
   commenceSingleSimulation: ->
     @listenToOnce @god, 'infinite-loop', @handleSingleSimulationInfiniteLoop
     @listenToOnce @god, 'goals-calculated', @processSingleGameResults
-    @god.createWorld {spells: @generateSpellsObject()}
+    @generateSpellsObject()
+        .then(((spell) => @god.createWorld {spells: spell}).bind(@))
```

Comment:
Same comment here. I think (and hope) that we can remove the `bind` and let the arrow function work.